src/components/shared/TreeView.js

```diff
@@ -257,16 +260,15 @@ type TreeViewProps = {
   appendageButtons: string[],
   disableOverscan: boolean,
   icons: IconWithClassName[],
-  contextMenu?: React$Element<*>,
+  contextMenu?: React.Element<any>,
```

Comment:
I'm nervous about `any`s, but I'm not sure I have a specific suggestion or idea on how this will affect our types in practice.

Reply:
In this case, it just says you accept any React element, you don't care because you just insert it as markup.